app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -378,15 +376,9 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
                 promptsStyling = DownloadsFeature.PromptsStyling(
                     gravity = Gravity.BOTTOM,
                     shouldWidthMatchParent = true,
-                    positiveButtonBackgroundColor = ThemeManager.resolveAttribute(
-                        R.attr.accent,
-                        context
-                    ),
-                    positiveButtonTextColor = ThemeManager.resolveAttribute(
-                        R.attr.contrastText,
-                        context
-                    ),
-                    positiveButtonRadius = (resources.getDimensionPixelSize(R.dimen.tab_corner_radius)).toFloat()
+                    positiveButtonBackgroundColor = context.getColorFromAttr(R.attr.accent),
```

Comment:
So I seem to remember that we are doing this differently in some places for a reason... They're not equivalent depending on what the receiver is expecting. Ie look at these right next to each other: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt#L48-L51

If we want to change these, please test thoroughly.

Reply:
I've tested this helper and I wrote it to move `ThemeManager.resolveAttribute` into AC. The implementation is exactly the same so it should work fine.

Reply:
Wait scratch that, I'm confusing `getColor` with `resolveAttribute`. I'll fix it here.